### ISSUE TITLE
Downloading a folder doesn't include subfolders

### ISSUE THREAD

**Issue by frankrousseau** [cozy/cozy-files#407]
I wanted to backup my files before using `cozy-desktop` but I wasn't able to download my files. Subfolders of my favorite folders were not included in my download.

**Comment by LaPalice** [comment]
Same problem here + Files tells me my files have been transfered which is far worse because some people will not check twice

**Comment by frankrousseau** [comment]
What do you mean by "Files tells me my files have been transfered"?

**Comment by LaPalice** [comment]
Hm, I might have misread your issue. I was talking about the app Files. Maybe I should move my ticket ?

**Comment by frankrousseau** [comment]
No it's ok, it's just I didn't understand what you meant. But after reading it again, It looks clearer to me. You mean that in some way the download failed but the app doesn't tell you that something went wrong. Am I right?

**Comment by LaPalice** [comment]
Yes, to be more precise, I tried to copy a folder, the app told me "Your files have been copied" (or something like that" but in the end, nothing has been copied.
